client/src/components/molecules/NetworkSwitcher.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState, useContext, useEffect, useRef } from 'react'
+import { urlq } from '../../utils/utils'
+import { CONNECTIONS } from '../../config'
+import { User } from '../../context'
+import styles from './NetworkSwitcher.module.scss'
+
+const networkUrlParam = urlq.get('network') || ''
+
+const getNetworkConfig = (network: string) => {
```

Comment:
this is hard to get what's going on I this function. I think the problem described in the code comment could be addressed with something like:

```js
const getNetworkConfig = (network = 'pacific') => {
    return (CONNECTIONS as any)[network]
}
```

Reply:
Tried to refactor it, but then this (below) gives me a problem with value (spent about an hour on it with no result even with the altorros guys' help) 
As typescript is being confused about the return type

>             <tbody>
                {Object.entries(oceanConfig).map(([key, val]) => (
                    <tr key={key}>
                        <td>
                            <code className={styles.label}>{key}</code>
                        </td>
                        <td>{val}</td>
                    </tr>
                ))}
            </tbody>

Reply:
oh no, sorry for the time consuming misguidance, also no clue why `any` would be a problem down there in `VersionsTable`. So whatever works then  :-)

Reply:
I know it's weird, I'm still trying to get my head around typescript :)
What's is weirder is that when I use your suggestion and try to print `val` in VersionsTable in the following way `${val}}`(exactly like that), it works. The compiler works in a mysterious way ...